games-action/dave_gnukem/dave_gnukem-1.0.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="2D platform shooter inspired by Duke Nukem 1"
+DESCRIPTION="Dake Gnukem is a retro-style 2D scrolling platform shooter
+similar to, and inspired by, Duke Nukem 1.
+
+The year is \$CURRENTYEAR+8.
+An evil genius, Dr Proetton, has been hired by the CIA
+to infect the world's computers with a virus called SystemD, crippling them.
+Only you can stop him. You must find the floppy disk with the
+Devuan Antivirus on it, and install it on the master computer,
```

Comment:
Removed to many backslashes :) Check the wiki page linked earlier

Reply:
I think I now understand what you meant. Is it better now?